mod/beacon/blockchain/execution_engine.go

```diff
@@ -72,7 +75,7 @@ func (s *Service[
 	}
 
 	prevBlockRoot := blk.HashTreeRoot()
-	if _, err := s.lb.RequestPayloadAsync(
+	if _, err = s.localBuilder.RequestPayloadAsync(
```

Comment:
future PR, but is there a clean way to make this async? (as well as the process slots above).